core/workspace_svg.js

```diff
@@ -1497,6 +1498,58 @@ Blockly.WorkspaceSvg.prototype.scrollCenter = function() {
   this.scrollbar.set(x, y);
 };
 
+/**
+ * Scroll the workspace to center on the given block.
+ * @param {?string} id ID of block center on.
+ * @public
+ */
+Blockly.WorkspaceSvg.prototype.centerOnBlock = function(id) {
+  if (!this.scrollbar) {
+    console.warn('Tried to scroll a non-scrollable workspace.');
+    return;
+  }
+  var block = this.getBlockById(id);
+  if (block) {
```

Comment:
last nit. Since the entire rest of the function is inside this if, perhaps returning early if the block doesn't exist is more obvious?